packages/components/pages/f-account-info/test-utils/component-objects/f-account-info-selectors.js

```diff
@@ -0,0 +1,46 @@
+exports.COMPONENT = '[data-test-id="account-info"]';
+exports.CHANGE_EMAIL_ADDRESS_LINK = '[name="account-info-change-emailAddress-link"]';
+exports.SAVE_CHANGES_BUTTON = '[data-test-id="account-info-save-changes-button"]';
+exports.CHANGE_PASSWORD_BUTTON = '[data-test-id="account-info-change-password-button"]';
+exports.DELETE_ACCOUNT_LINK = '[name="account-info-delete-account-link"]';
+
+
+exports.FIELDS = {
+    firstName: {
+        input: '[data-test-id="formfield-account-info-consumer-firstName-input"]',
+        emptyError: '[data-test-id="consumer-firstName-empty-error"]',
+        invalidError: '[data-test-id="consumer-firstName-invalid-error"]'
+    },
+    lastName: {
+        input: '[data-test-id="formfield-account-info-consumer-lastName-input"]',
+        emptyError: '[data-test-id="consumer-lastName-empty-error"]',
+        invalidError: '[data-test-id="consumer-lastName-invalid-error"]'
+    },
+    emailAddress: {
+        input: '[data-test-id="formfield-account-info-consumer-emailAddress"]'
```

Comment:
Have we catered for invalid / empty emails in the component?

Reply:
It's not possible to have an invalid/empty email in this component - the input box is completely disabled with the user's email inside -
![image](https://user-images.githubusercontent.com/49618712/148991146-ea5fc1f5-0e12-40a5-9c29-9dd2e9897059.png)